src/System.ComponentModel.TypeConverter/tests/BindingListTests.cs

```diff
@@ -0,0 +1,681 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// See the LICENSE file in the project root for more information. 
+
```

Comment:
I wanted to have a visual differentiator between the license header and the slightly different copyright line added.